### PR TITLE
feat(ui): added issue status modal

### PR DESCRIPTION
**Pull request by NandagopanP**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### Attach Screenshots For Reference

Kindly attach screenshot of UI changes made.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -13,12 +13,13 @@
...
   "dependencies": {
     "@chakra-ui/next-js": "^2.0.1",
-    "@chakra-ui/react": "^2.5.1",
+    "@chakra-ui/react": "^2.6.0",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "@fontsource/inter": "^4.5.15",
     "@hookform/resolvers": "^2.9.11",
     "@next/font": "13.1.6",
+    "@ramonak/react-progress-bar": "^5.0.3",
```
Comment (sreehari2003) [inline web/package.json:22]: what is this ?

Comment (NandagopanP) [reply web/package.json:22]: i used that for importing react progress bar

Comment (sreehari2003) [reply web/package.json:22]: where are you using this progress bar ?

Comment (NandagopanP) [reply web/package.json:22]: i dint use it

Comment (sreehari2003) [reply web/package.json:22]: image

you only updated chakra ui and added the modal why too much file changes ?

Comment (sreehari2003) [reply web/package.json:22]: > i dint use it

pinne enthina nee install akie ?

File: web/package-lock.json
Comment (sreehari2003) [inline web/package-lock.json:1]: we are using yarrn so remove this from git

**Review by sreehari2003 — APPROVED** [review]

**Pull request merged**